scala-collector/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM java:8-alpine
+
+ENV SCALA_COLLECTOR=20170611 \
+BUILD_PACKAGES='bash curl-dev git curl ca-certificates gettext' \
+SBT_VERSION=0.13.15 \
```

Comment:
Since [the SSC still uses sbt 0.13.0](https://github.com/snowplow/snowplow/blob/master/2-collectors/scala-stream-collector/project/build.properties) at the moment, would it be possible to change this version accordingly?

Otherwise, sbt will just end up downloading 0.13.0 when launched.